scripts/build/build_darwin_framework.py

```diff
@@ -114,6 +124,8 @@ def build_darwin_framework(args):
                         help="Output log file destination",
                         required=True)
     parser.add_argument('--ipv4', action=argparse.BooleanOptionalAction)
+    parser.add_argument('--asan', action=argparse.BooleanOptionalAction)
+    parser.add_argument('--tsan', action=argparse.BooleanOptionalAction)
```

Comment:
Nothing above seems to be doing anything with the tsan option...

Reply:
Fixed.